app/code/Magento/Sitemap/Api/Data/SitemapExtensionInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\Sitemap\Api\Data;
+
+/**
+ * Sitemap Extension interface
+ */
+interface SitemapExtensionInterface extends \Magento\Framework\Api\ExtensionAttributesInterface
```

Comment:
Please, remove this interface, it will be generated automatically.

Reply:
Removed